static/app/views/onboarding/createSampleEventButton.spec.jsx

```diff
@@ -22,7 +22,8 @@ describe('CreateSampleEventButton', function () {
         project={{...project, platform: 'javascript'}}
       >
         {createSampleText}
-      </CreateSampleEventButton>
+      </CreateSampleEventButton>,
+      {organization: org}
```

Comment:
This was the only broken test, it needs to be passed the org so it is properly inserted into the legacy context